apps/dashboard/src/components/workflow-editor/steps/email/email-tabs.tsx

```diff
@@ -55,10 +56,16 @@ export const EmailTabs = (props: StepEditorProps) => {
       </header>
       <Separator />
       <TabsContent value="editor" className={tabsContentClassName}>
-        <EmailEditor uiSchema={uiSchema} />
-        <EmailTabsEditSection>
-          <CustomStepControls dataSchema={dataSchema} origin={workflow.origin} />
-        </EmailTabsEditSection>
+        {workflow.origin === WorkflowOriginEnum.NOVU_CLOUD && (
```

Comment:
Show sections based on workflow type avoiding redundant empty divs or the dangling Email Acme header